components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -137,7 +137,7 @@ class BrowserIcons(
      */
     fun install(engine: Engine, store: BrowserStore) {
         engine.installWebExtension(
-            id = "mozacBrowserIcons",
+            id = "icons@mozac.org",
```

Comment:
Is there any specifics about changing the ID? As the ext are persisted, doing that could duplicate the extension on devices that already have this ext?

Reply:
No problem, I saw the comments !

Reply:
👍 Yeah, new ID format is a requirement. Previous API was transient so extension isn't persisted yet. Going forward, we can't change IDs easily, not without landing some migration code that first removes the old one...